SECURITY.md

```diff
@@ -0,0 +1,55 @@
+# Securing the dashboard
+
+The following page describes some ways to secure the dashboard.
+
+## Basic Authentication on Istio with Envoy Filter
+When you are using Istio there is a way of introducing an envoy filter which contains
```

Comment:
Which Istio version did you use and how was it installed? There are plenty of versions out there so let's add an example working config (so which kubectl apply command to do, or istioctl - that kinda business)